### PR TITLE
Wait for ajax

### PR DESCRIPTION
**Pull request by rizzza**
Adding wait_for_ajax helper in `Page`.

### CODE REVIEW

File: lib/page.rb [hunk 1]
```diff
@@ -77,6 +77,20 @@ def self.execute_script(script)
       Driver.execute_script_driver(script)
     end
 
+    def self.evaluate_script(script)
```
Comment (sethuster) [inline lib/page.rb:80]: We have this functionality available in the driver object.

File: lib/page.rb [hunk 2]
```diff
@@ -77,6 +77,20 @@ def self.execute_script(script)
...
+      Driver.evaluate_script script
+    end
+
+    def self.wait_for_ajax
```
Comment (sethuster) [inline lib/page.rb:84]: would this make more sense in the driver object?

File: lib/page.rb (hunk 1 above)
Comment (rizzza) [inline lib/page.rb:80]: > We have this functionality available in the driver object.

I was going with the convention already implemented [here](https://github.com/rizzza/gridium/blob/a9d193ef8add83d8d1946012277bd54e7b96897f/lib/page.rb#L76-L78)

To me, it makes sense to have it available in the `Page` object encapsulating the `Driver`.

File: lib/page.rb (hunk 2 above)
Comment (rizzza) [inline lib/page.rb:84]: > would this make more sense in the driver object?

To me, it makes sense to have it available in the Page object encapsulating the Driver.

IMHO, it reads better. I want to `wait_for_ajax` on the `Page` object.

Comment (sethuster) [inline lib/page.rb:84]: Fair enough.

**Comment by sethuster** [comment]
LGTM

**Pull request merged**